scripts/notify_team_new_comment.js

```diff
@@ -0,0 +1,108 @@
+const fs = require('fs');
+const path = require('path');
+const { LE_BOT_USERNAME, BOT_MESSAGE } = require('./constants');
+const { CLOSE_CONTRIBUTORS } = require('./close_contributors');
+
+module.exports = async ({ github, context, core }) => {
+  try {
+    const issueNumber = context.payload.issue.number;
+    const issueUrl = context.payload.issue.html_url;
+    const issueTitle = context.payload.issue.title;
+    const escapedTitle = issueTitle.replace(/"/g, '\\"');
+    const commentId = context.payload.comment.id;
+    const commentTime = new Date(context.payload.comment.created_at);
+    const oneHourBefore = new Date(commentTime - 3600000);
+    const commentAuthor = context.payload.comment.user.login;
+    const commentBody = context.payload.comment.body;
+    const repo = context.repo.repo;
+    const owner = context.repo.owner;
+    const slackWebhookUrl = process.env.SLACK_WEBHOOK_URL;
+    const communityWebhookUrl = process.env.SLACK_COMMUNITY_NOTIFICATIONS_WEBHOOK_URL;
+    const keywordsPath = path.join(__dirname, 'keywords.txt');
+    const keywordRegexes = fs.readFileSync(keywordsPath, 'utf-8')
+      .split('\n')
+      .map(k => k.trim().toLowerCase())
+      .filter(Boolean)
+      .map(keyword => new RegExp(`\\b${keyword}\\b`, 'i'));
+
+
+
+    async function hasLabel(name) {
+        let labels = [];
+        try {
+          const response = await github.rest.issues.listLabelsOnIssue({
+            owner,
+            repo,
+            issue_number: issueNumber
+          });
+          labels = response.data.map(label => label.name);
+        } catch (error) {
+           core.warning(`⚠️ Failed to fetch labels on issue #${issueNumber}: ${error.message}`);
+           labels = [];
+        }
+        return labels.some(label => label.toLowerCase() === name.toLowerCase());
+    }
+
+    async function findRecentCommentsByUser(username) {
+      try{
+          let response = await github.rest.issues.listComments({
+              owner,
+              repo,
+              issue_number: issueNumber,
+              since: oneHourBefore.toISOString()
+          });
+          return response.data.filter(comment => comment.user.login === username);
+      } catch (error) {
+          core.warning(`⚠️ Failed to fetch comments on issue #${issueNumber}: ${error.message}`);
+          return [];
+      }
+    }
+
+    async function botReply(){
+        let response = null;
+        try {
+            response = await github.rest.issues.createComment({
+              owner,
+              repo,
+              issue_number: issueNumber,
+              body: `Hi @${commentAuthor} ${BOT_MESSAGE}`
```

Comment:
Again, perhaps a maliciously formed author username might be a concern here? But again, I think we have to rely on Github having not allowed such things.

Reply:
Yes, hopefully. Nothing better comes to my mind now either. Thanks for pointing this place.